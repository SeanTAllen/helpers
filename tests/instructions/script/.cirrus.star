load("../../../lib.star", "task", "container", "script")

def main(ctx):
    return [
        task("script instruction with single command", container(), instructions=[
            script("single", "echo \"step 1\"")
        ]),
        task("script instruction with multiple commands", container(), instructions=[
            script("multiple", "echo \"step 1\"", "echo \"step 2\"")
        ]),
    ]
