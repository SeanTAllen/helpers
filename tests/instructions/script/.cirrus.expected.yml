task:
  name: script instruction with single command
  container:
    cpu: 2
    memory: 4096
  single_script:
    - echo "step 1"
task:
  name: script instruction with multiple commands
  container:
    cpu: 2
    memory: 4096
  multiple_script:
    - echo "step 1"
    - echo "step 2"
